test/gui/shared/steps/account_context.py

```diff
@@ -29,6 +29,9 @@ def step(context):
 
 @Then('the account with displayname "|any|" and host "|any|" should be displayed')
 def step(context, displayname, _):
+    if get_config('predefined_users'):
+        # get displayname from first name
+        displayname = get_displayname_for_user(displayname.split()[0].strip())
```

Comment:
maybe need to lowercase the username?
```suggestion
        displayname = get_displayname_for_user(displayname.split()[0].strip().lower())
```